R/module_teal.R

```diff
@@ -136,6 +138,30 @@ srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
       title = "SessionInfo"
     )
 
+    output$lockFile <- downloadHandler(
```

Comment:
@pawelru what if we put this downloadHandler into a separate function that is also included in `R/teal_lockfile.R`. This way all functionality of lockfile is in one R file : )

Reply:
good idea 👍

Reply:
a6c3691e054868f1baef51d3e50f102f4f8a6eeb